Taskfile.yml

```diff
@@ -75,51 +68,71 @@ tasks:
       - go build -ldflags "{{.LDFLAGS}}" ../cmd/out
       - go build -ldflags "{{.LDFLAGS}}" ../cmd/hello
 
-  docker-login:
-    silent: true
+  build:copydir:
```

Comment:
For my understanding: How iand why is this copydir binary used?

Reply:
It is useful when developing `help.CopyDir()` and doing manual tests.

Function `help.CopyDir()` is the equivalent of `afero` in fly_helper, but is only one function and transparent to the production code: the code uses the real filesystem.

`help.CopyDir()` uses Go template to generate any kind of input file for testing.

Best way to understand is to see how the tests use `help.CopyDir()`.